### PR TITLE
fix(monitoring): fix host and service status detail page, mostly ergonomics

### PR DESCRIPTION
**Pull request by cgagnaire**
+ Fix some ergonomics on those pages to make them look cleaner,
+ Fix graph button on 'Host > Performances',
+ Add the handling of services without any graph (hide "Detailed Graph" section).

### CODE REVIEW

File: www/include/monitoring/objectDetails/template/hostDetails.ihtml [hunk 1]
```diff
@@ -367,10 +371,10 @@
                 </tr>
             </table>
             <br />
-            <table class="table ListTable linkList">
+            <table class="ListTable">
                 <tr class='list_lvl_1'>
                     <td class="FormHeader" style="width: 50%;border-bottom:1px solid #d1d2d4;">
-                        <h4>{t}Host commands and shortcuts{/t}</h4>
+                        <h4>{t}{$m_mon_host_commands_shortcuts}{/t}</h4>
```
Comment (kduret) [inline www/include/monitoring/objectDetails/template/hostDetails.ihtml]: {t} is not needed anymore cause translation is done before

Comment (cgagnaire) [reply www/include/monitoring/objectDetails/template/hostDetails.ihtml]: Hopefully I only did this on that one :)

**Comment by cgagnaire** [comment]
For info, just did a rebase to use correct commit comment

**Comment by cgagnaire** [comment]
Before : 
service_detail_before
After : 
service_detail_after

**Comment by cgagnaire** [comment]
Before : 
host_detail_before
After : 
host_detail_after

**Comment by lpinsivy** [comment]
@cgagnaire can you base your PR on 18.10.X/master branches?

**Comment by cgagnaire** [comment]
Too old :(

**Pull request closed without merging**